lib/v2/screens/explore_screens/vote_screens/vote/interactor/viewmodels/proposal_type_model.dart

```diff
@@ -13,23 +13,17 @@ class ProposalType {
 }
 
 const List<ProposalType> proposalTypes = [
-  ProposalType(
-    type: 'Stage',
-    stage: 'staged',
-    status: ['open'],
-    isReverse: false,
-  ),
   ProposalType(
     type: 'Open',
     stage: 'active',
     status: ['open'],
     isReverse: false,
   ),
   ProposalType(
-    type: 'Evaluate',
-    stage: 'active',
-    status: ['evaluate'],
-    isReverse: true,
+    type: 'Upcoming',
+    stage: 'staged',
+    status: ['open'],
+    isReverse: false,
```

Comment:
Also idk if this class and const list count as view_models or should be in local models folder 😕

Reply:
Yea this proposals items from the server have stage and status fields, This Object List contains the filters for each section for those fields and an extra field to sort the list.